I ocr implementieren

schmitzdominic opened the pull request:
- Google Vision implementiert
- Standard Kamera angebunden
- Extra Activity zum debugging eingefügt (A_Show_Recognition)

schmitzdominic committed:
Branch i_ocr_implementieren erstellt.
startActivitiy umbenannt in startActivity

schmitzdominic committed:
OCR implementiert ERSTE VERSUCHE!.

schmitzdominic committed:
OCR implementiert Zweiter Versuch!.

schmitzdominic committed:
OCR debugg

schmitzdominic committed:
OCR Image View

schmitzdominic committed:
OCR implementierung TEST 3

schmitzdominic committed:
Formater added

schmitzdominic committed:
Leerzeichen entfernt

schmitzdominic committed:
Permissions added

schmitzdominic committed:
Zahlen Logik Versuch 1

schmitzdominic committed:
Zahlen Logik Versuch 2

schmitzdominic committed:
Zahlen Logik Versuch 3

schmitzdominic committed:
Zahlen Logik Versuch 4

schmitzdominic committed:
Zahlen Logik Versuch 4

schmitzdominic committed:
Zahlen Logik Versuch 5

schmitzdominic committed:
Gradle build changed

schmitzdominic committed:
Rules added.

schmitzdominic committed:
Code dokumentiert und weitere Regeln hinzugefügt.

schmitzdominic committed:
Unused Code removed

joeymez reviewed (approved):
Einige Sachen kommen wir neu vor, aber der größte Teil ist verständlich. Freigabe durch mich gewährleistet! :)

dontcallme reviewed (approved):
Hab mal n weng probiert, gibt das aus was im code steht, scheint also zu funktionieren, versteh noch nicht alles, sieht aber gut soweit aus und geb auch mein ok, gut gemacht :)

schmitzdominic committed:
- Google Vision geupdatet
- Kategorie in der strings.xml für A_Show_Recognition angelegt
- Klasse S, out Methoden Dokumentiert.

Sippl reviewed (approved):
Läuft bei mir einwandfrei.  Permissions lassen sich vergeben. Kamera lässt sich öffnen, Bild lässt sich machen und speichern. Paar stellen im Code kann ich leider nicht evaluieren, weil ich nicht genau weiß was es macht. Jedoch im größtenteil verständlich. Die offenen Fragen / Punkte kläre ich direkt mit Domi, Soweit kann ich den reqeust approven

Sippl reviewed (commented):
Mir ist noch nen Bug aufgefallen, wenn ich die Permissions verweigere und dennoch kamera aufrufen möchte. Stürzt bei mir die APP ab (Bonprix angehalten).

schmitzdominic commented:
Ja das müssen wir noch implementieren, vielleicht wenn man verweigert eine Meldung "App kann nicht ausgeführt werden ohne Berechtigungen" o.ä. und die app sich dann schließt.

schmitzdominic committed:
C_Rules verbessert.
- getPLZ optimiert und angepasst
- getPrices komplett überarbeitet und verbessert
